dashboard/app/views/devise/registrations/_finish_sign_up.html.haml

```diff
@@ -21,8 +21,10 @@
       .field
         = f.label :user_type, t('signup_form.create_user_type')
         = f.select :user_type, user_type_options, {include_blank: true}, {disabled: @user.should_disable_user_type?}
+      - if @user.errors[:user_type].present?
+        %span.error.padded= t('activerecord.errors.messages.blank')
```

Comment:
Manually setting the error here is yucky, but that's because the ActiveRecord error ("is not included in the list") is very confusing for users.